### PR TITLE
Updated Installation Guide

### PR DESCRIPTION
**Pull request by vidhanarya**
Updated installation guide for installation in Windows/macOS based system as the previous guide didn't take into account that 'pykep' has issues on macOS and Windows. It'll help users with Issue #149

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,55 +9,54 @@
 __orbitdeterminator__ is a package written in python3 for determining orbit of satellites based on positional data. Various filtering and determination algorithms are available for satellite operators to choose from.  
 
 ### Installation Instructions
-Run the following commands to install orbitdeterminator:
 
+##### Virtual environment setup
+For this guide we will be using **miniconda**
+**Step 1**: Install miniconda using script and instruction [here](https://docs.conda.io/en/latest/miniconda.html)
+**Step 2**: After installation, create a virtualenv with python3.5 as
```
Comment (aryadas98) [inline README.md]: Small problem: all steps are appearing on same line

Comment (vidhanarya) [reply README.md]: Sorry I was busy these last two days. I did checked the code on online markdown editor before pushing, I'll fix and push a new commit

**Comment by aryadas98** [comment]
Is it possible to install this easily without conda? We don't want to add another "dependency" unless it's really required.

**Comment by vidhanarya** [comment]
This is not a dependency of the project but issue in pykep package so I believe unless pykep maintainers fix the issue for macOS and Windows, we have to use the steps mentioned in the updated readme.md. Besides, it is always a good practice to use different environments for different kind of projects and conda helps us with that.

**Comment by aryadas98** [comment]
Yes, I know. Is it not possible to use just virtualenv though?

**Comment by vidhanarya** [comment]
Then we will need to build pykep from source. Should I change the mentioned solution in Readme.md as a workaround and main guide as using virtualenv and instruction for building pykep from source??

**Comment by aryadas98** [comment]
No, we want orbitdeterminator to be easy to install. So let this be the main method but also write the method for those who don't want to use conda. :)

**Comment by aryadas98** [comment]
Yes this is good. :+1:

**Pull request merged**